wrapper.ts

```diff
@@ -118,15 +120,26 @@ export class Pty {
   }
 
   close() {
-    this.#socket.destroy();
+    this.#socket.end();
```

Comment:
huh, maybe it's worth adding a comment as to why we chose `end` instead of `destroy`. there are subleties here, haha.